app/policies/supported_permission_policy.rb

```diff
@@ -0,0 +1,16 @@
+class SupportedPermissionPolicy < BasePolicy
+  class Scope < ::BasePolicy::Scope
+    def resolve
+      if current_user.superadmin?
+        scope.all
+      elsif current_user.admin?
+        scope.all
+      elsif current_user.organisation_admin?
+        app_ids = Pundit.policy_scope(current_user, :manage_supported_permissions_for_application).pluck(:id)
```

Comment:
Gone with `UserPermissionManageableApplicationPolicy`.  `UserPermission` because I wanted to be clear that it wasn't the ability to add/remove permissions from the application (e.g. you shouldn't can't use this scope to get the list of applications that you are allowed to edit).  `Manageable` rather than `Grantable` because a) we can revoke permissions as well as grant them, b) there's a method `manage_supported_permissions?` on the `UserPolicy` that is used to say if a user is allowed to do this kind of thing at all, and I hoped that the `Managable` would act as a link to the `manage` part of that.